attributes/default.rb

```diff
@@ -1,26 +1,57 @@
-default['openstack-mistral']['install_recipe'] = 'openstack-mistral::install_source'
+# The home is set to null by package
+default['openstack-mistral']['home'] = nil
```

Comment:
Is this mandatory?
Not used anywhere, or it's some kind of "special" Chef setting?